### ISSUE TITLE
Trojan detected

### ISSUE THREAD

**Issue by mazenkhalil** [patterns-ai-core/langchainrb#683]
The following file has been detected as Trojan!

`spec/fixtures/loaders/clearscan-with-image-removed.pdf`

![image](https://github.com/patterns-ai-core/langchainrb/assets/21160900/8deb95fc-13c0-41c4-8550-93d2b862fdb0)

**Comment by andreibondarev** [comment]
I'm really doubtful that @rickychilcott would've added this file with a malicious intent.

**Comment by mazenkhalil** [comment]
It has been detected by Norton on Mac OS. I have submit the file for further analyses.

![image](https://github.com/patterns-ai-core/langchainrb/assets/21160900/abea6912-434c-4c73-807c-3c22669fa678)

**Comment by rickychilcott** [comment]
Definitely not intentional if there is an issue there. I can't remember where I sourced this file, but if I remember correctly, I think I found a set of "pdf samples" from a university. But I can't be sure and can't find them with a Google search.

I must admit, I'm not feeling very confident about the file source.

I did try that file on [VirtusTotal](https://www.virustotal.com/) - see the scan results for [clearscan-with-image-removed.pdf](https://www.virustotal.com/gui/file/9d8e3994e70d41ebc1bb0ecdb0fdaf133c0d6701f743e933f9666b9f40faa4e7) and https://internxt.com/virus-scanner and both seem to say it's not an issue. But... they might not have the signatures.

These can be swapped easily if there is concern and I apologize if there is indeed an issue.

**Comment by andreibondarev** [comment]
@mazenkhalil Could it be that your local copy of the file is compromised? I would try pulling it down again and running through a scan.

**Comment by mazenkhalil** [comment]
I have just pulled a new version from the repository, and the file has been immediately quarantined. As I mentioned earlier, I have submitted a report to Norton for further analysis. Whether the file is actually infected or it is a false alarm, I recommend acting quickly and replacing it with something else.

![image](https://github.com/patterns-ai-core/langchainrb/assets/21160900/47ab4a56-5eb2-4aff-bbda-196305dc5e21)
